docs/_includes/tk-mari

```diff
@@ -1 +1 @@
-4.0 — 4.7
\ No newline at end of file
+4.0 — 4.8
```

Comment:
I wonder how we apply the "current + 2 previous ones" rule for this one.
Any idea?

Reply:
Maybe supporting the previous 2 minors? Like 4.6 - 4.8

Reply:
@mathurf any advice?